cpp/ql/src/Critical/NotInitialised.ql

```diff
@@ -11,6 +11,7 @@
 import cpp
 
 // See also InitialisationNotRun.ql and GlobalUseBeforeInit.ql
```

Comment:
If this was a multi-line `/* .. */` comment it would get the extra newline after it.  Generally the autoformatter expects toplevel comments to be of the form `/* .. */` and in-predicate comments to be `// ...`.

Reply:
I personally tend to use `//` for most comments, unless either (1) it's going to be many lines or (2) I want to finish it before the end of the line.  Of course use of qldoc comments `/** .. */` frequently overrides this preference.

Reply:
(in this case I believe the comments were originally written by someone else)